### PR TITLE
Adding a link in the screenshots section to the screenshots GitHub issue

### PR DESCRIPTION
**Pull request by aajjbb**
As the title says per se.

### CODE REVIEW

File: screenshots.mdwn [hunk 1]
```diff
@@ -1,8 +1,5 @@
...
 
-<div id="screenshot">
-<figure>
-    [[!img images/screen.png alt="Screenshot 1" size="382x216"]]
-    <figcaption>awesome v4.x + lain + multicolor theme</figcaption>
-</figure>
-</div>
+Screenshots of different AwesomeWM configurations are available [here](https://github.com/awesomeWM/awesome/issues/1395).
+
+Feel free to add your own Screenshot in the thread along with it's configuration details.
```
Comment (blueyed) [inline screenshots.mdwn]: *screenshot
*its

Comment (aajjbb) [inline screenshots.mdwn]: Sorry for the typos, fixed.

**Comment by blueyed** [comment]
Thanks, cherry-picked and slightly amended.

**Comment by blueyed** [comment]
Will likely not show up though for now: https://github.com/awesomeWM/awesome-www/issues/89.

**Pull request closed without merging**